trusted-committer/07-becoming-a-trusted-committer.asciidoc

```diff
@@ -1,68 +1,45 @@
+In grassroots communities, the founders often assume the role of the Trusted Committer. 
+As a community grows, or in larger communities, Trusted Committers are usually nominated or voted in from the community https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[contributors].
+But the Trusted Committer role should be taken on voluntarily as it requires an immense amount of time and dedication to be successful in it.
+
+What are the criteria to apply in nominating https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[contributors] for a Trusted Committer role? 
+What does it take to successfully fill the role of a Trusted Committer? 
+First off, potential Trusted Committers need to have demonstrated a deep technical competence during their work in the community. 
+In addition to that, they must have proven their ability to effectively communicate with peers in the community and ideally also with
+product owners and with management.
+
+In the same vein, they must have shown the willingness and patience to use their skills and spend intentional time upleveling contributors.
+Finally, fulfilling the Trusted Committer role requires a certain emotional maturity to deal with stressful social situations, which are
+bound to come up from time to time. 
+Contributors who satisfy these criteria will be good potential Trusted Committers, in our opinion.
+
+The Trusted Committer role might not appear all that attractive to some contributors as it means spending less time coding. 
+Being nominated for the Trusted Committer role might even be perceived by some as a demotion or as negative feedback on their coding skills. 
+But the opposite is true. 
+Being nominated for the Trusted Committer role usually means someone has recognized your valuable contributions and sees in you the potential to grow and lead. 
+The Trusted Committer role will give you more influence over the evolution of the codebase and will ultimately make you a more complete
+developer. 
+Explaining to contributors how the software works more often than not leads to new insights on part of the Trusted Committer and will help to identify opportunities to improve the software.
+
+Whether you have one or multiple Trusted Committers depends on the size and the risk associated with the software developed by the InnerSource community. 
+The Trusted Committer role is time consuming, and not everyone is willing or able to make that type of commitment. 
+Because of this, some companies use a _Trusted Committer rotation_ system where multiple Trusted Committers share the workload of the Trusted Committer role, and the Trusted Committers who are not _on duty_ can exclusively focus on tech-oriented work. 
+Having more than one Trusted Committer also makes it easier when someone leaves the company or moves on from the role to do something else.
```

Comment:
I actually liked having the "In that case, it is important	
that there are other Trusted Committers in place already, who can take over and ensure	
continuity in the community." sentence as additional part here.

Reply:
added back